media-sound/cadence/cadence-9999.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit git-r3 python-single-r1 xdg desktop
+
+DESCRIPTION="Collection of tools useful for audio production"
+HOMEPAGE="http://kxstudio.linuxaudio.org/Applications:Cadence"
+EGIT_REPO_URI="https://github.com/falkTX/Cadence.git"
+KEYWORDS=""
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="a2jmidid ladish -pulseaudio opengl"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/dbus-python[${PYTHON_MULTI_USEDEP}]
+		dev-python/PyQt5[dbus,gui,opengl?,svg,widgets,${PYTHON_MULTI_USEDEP}]
+	')
+	media-sound/jack2[dbus]
+	media-sound/jack_capture
+	a2jmidid? ( media-sound/a2jmidid[dbus] )
+	ladish? ( >=media-sound/ladish-9999 )
+	pulseaudio? ( media-sound/pulseaudio[jack] )
+"
+RDEPEND="${CDEPEND}"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	sed -i -e "s/python3/${EPYTHON}/" \
+		data/cadence \
+		data/cadence-aloop-daemon \
+		data/cadence-jacksettings \
+		data/cadence-logs \
+		data/cadence-render \
+		data/cadence-session-start \
+		data/catarina \
+		data/catia \
+		data/claudia \
+		data/claudia-launcher || die "sed failed"
+
+	default
+}
+
+src_compile() {
+	myemakeargs=(PREFIX="${EPREFIX}/usr"
+		SKIP_STRIPPING=true
+	)
+
+	emake "${myemakeargs[@]}"
+}
+
+src_install() {
+	emake PREFIX="${EPREFIX}/usr" DESTDIR="${ED}" install
```

Comment:
I guess we need `${ED}` here instead of `${D}` because there's no configure for cadence because it's Makefile/install only?